examples/bri-3/.env.sample

```diff
@@ -16,5 +16,7 @@ BPI_NATS_SERVER_USER="bpi_operator"
 BPI_NATS_SERVER_PASS="liftboiliftboiliftboiliftboi1"
 BPI_ENCRYPTION_KEY_K_PARAM="yzkXp3vY_AZQ3YfLv9GMRTYkjUOpn9x18gPkoFvoUxQ"
 BPI_ENCRYPTION_KEY_KTY_PARAM="oct"
+SNARKJS_CIRCUIT_KEYS_PATH="zeroKnowledgeKeys/circuit/"
```

Comment:
I think we can remove these variables as the paths are autogenerated by the snarkjs library when the `npm run snarkjs:circuit {circuitName}` script is executed.

Reply:
I was thinking in the direction that a workgroup admin would be required to provide a compiled circuit and all the necessary artifacts during the setup phase. They would generate those in their own environment and just place it in a predetermined folder where the BPI has access to it, and would have to follow a predefined naming convention.

This is why i think it makes sense to have the parh as part of configuration, because the BPI admin would be able to choose where to store the artifacts. If you agree, i would keep this config value with slight adjustments based on your inputs.

Reply:
Ok we can allow workgroup admins to provide compiled circuit and all the related artifacts. But for the demo, we would have to compile the circuits ourselves. For that purpose, we can use the code that I added.